eth/backend.go

```diff
@@ -1174,32 +1183,35 @@ func (s *Ethereum) StartMining(ctx context.Context, db kv.RwDB, stateDiffClient
 		return fmt.Errorf("etherbase missing: %w", err)
 	}
 
-	if borcfg == nil {
-		s.logger.Error("mining is not supported after the Merge")
-		return errors.New("mining is not supported after the Merge")
-	}
-
 	if miner.MiningConfig.Enabled {
+		if s.chainConfig.ChainName == networkname.DevChainName {
+			miner.MiningConfig.SigKey = core.DevnetSignPrivateKey
+		}
 		if miner.MiningConfig.SigKey == nil {
 			s.logger.Error("Etherbase account unavailable locally", "err", err)
 			return fmt.Errorf("signer missing: %w", err)
 		}
+		if borcfg != nil {
+			borcfg.Authorize(eb, func(_ libcommon.Address, mimeType string, message []byte) ([]byte, error) {
+				return crypto.Sign(crypto.Keccak256(message), miner.MiningConfig.SigKey)
+			})
 
-		borcfg.Authorize(eb, func(_ libcommon.Address, mimeType string, message []byte) ([]byte, error) {
-			return crypto.Sign(crypto.Keccak256(message), miner.MiningConfig.SigKey)
-		})
-
-		if !s.config.WithoutHeimdall {
-			err := stagedsync.FetchSpanZeroForMiningIfNeeded(
-				ctx,
-				s.chainDB,
-				s.blockReader,
-				borcfg.HeimdallClient,
-				logger,
-			)
-			if err != nil {
-				return err
+			if !s.config.WithoutHeimdall {
+				err := stagedsync.FetchSpanZeroForMiningIfNeeded(
+					ctx,
+					s.chainDB,
+					s.blockReader,
+					borcfg.HeimdallClient,
+					logger,
+				)
+				if err != nil {
+					return err
+				}
 			}
+		} else if s.chainConfig.Consensus == chain.CliqueConsensus {
+			s.engine.(*clique.Clique).Authorize(eb, func(_ libcommon.Address, _ string, msg []byte) ([]byte, error) {
+				return crypto.Sign(crypto.Keccak256(msg), miner.MiningConfig.SigKey)
+			})
 		}
```

Comment:
Plz add here (i.e. if not Bor and not Clique)
```
else {
  s.logger.Error("mining is not supported after the Merge")
  return errors.New("mining is not supported after the Merge")
}
```
to ensure that this old-school mining is not enabled for Ethereum mainnet & Gnosis Chain. See also Issue #10716.